docs/resources/dhcp_fixed_address.md

```diff
@@ -58,6 +58,13 @@ resource "bloxone_dhcp_fixed_address" "example_fixed_address" {
   depends_on = [bloxone_ipam_subnet.example]
 }
 
+resource "bloxone_dhcp_option_code" "option_code" {
+  code         = 250
+  name         = "example_option_code"
+  option_space = bloxone_dhcp_option_space.option_space.id
+  type         = "int32"
+}
+
```

Comment:
I would rather use a data source to get an existing Option Code (like for ex. option 6). This is a more common scenario.

Reply:
done